plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -156,12 +156,15 @@ open class GenerateGramineBundle @Inject constructor(
      * This will create a .manifest.sgx and a .sig files into the output dir.
      */
     private fun generateSgxManifestAndSigstruct() {
+        val signingKeyPath = linuxExec.prepareFile(signingKey.get().asFile.toPath())
+
         execCommand(
             "gramine-sgx-sign",
             "--manifest=${GRAMINE_MANIFEST}",
-            "--key=${signingKey.get().asFile.absolutePath}",
+            "--key=${signingKeyPath.absolutePathString()}",
             "--output=${GRAMINE_SGX_MANIFEST}"
         )
+        linuxExec.cleanPreparedFiles()
```

Comment:
It would be better if we surround it with `try/finally` block as we do elsewhere, e.g. `AddEnclaveSignature.kt`

Reply:
done, thanks!